pyfritzhome/devicetypes/fritzhomedevicelightbulb.py

```diff
@@ -58,6 +60,10 @@ def _update_lightbulb_from_node(self, node):
                     "supported_modes"
                 )
 
+                self.fullcolorsupport = colorcontrol_element.attrib.get(
```

Comment:
we should use the `self.get_node_value_as_int_as_bool()` method here, so we get a bool out of it

Reply:
We can't use `get_node_value_as_int_as_bool()` here, as this is not a node value but an attribute. But you are right, that it should be a `bool`.